docker/test/performance-comparison/compare.sh

```diff
@@ -1074,6 +1074,53 @@ wait
 unset IFS
 }
 
+function upload_results
+{
+    if ! [ -v CHPC_DATABASE_URL ]
+    then
+        echo Database for test results is not specified, will not upload them.
+        return 0
+    fi 
+
+    # Surprisingly, clickhouse-client doesn't understand --host 127.0.0.1:9000
```

Comment:
You can use `SocketAddress` for this purpose.
Just don't forget to test all the cases:

`127.0.0.1:9000`
`[2a02:6b8:a::a]:9000`
`[::]:9000`
`[::ffff:127.0.0.1]:9000`

Reply:
And also to propagate it to all clickhouse tools.